.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -47,12 +47,22 @@ Enter your extended release note in the block below. If the PR requires addition
 
 <!--
 This section can be blank if this pull request does not require a release note.
-Please use the following format for linking documentation or pass the
-section below:
+
+The Kubernetes Release Notes website (https://relnotes.k8s.io) is able to
+provide additional documentation to the end users, which should help them to
+retrieve further documentation resources about the change in this PR.
+
+If you decide to add links which point to resources within git repositories,
+please ensure that the appropriate revision is included in the link and not
+generic ones like `master`. The same applies to external documentation, which
+may be not available any more because they got updated to a more recent version.
+
+Beside this, please use the following format for linking documentation:
```

Comment:
I'd just nix "Besides this, " and start with "Please use the following"

Reply:
Alright, adapted as suggested.